browser/utils/android/java/src/org/chromium/brave/browser/utils/DrawableUtils.java

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.brave.browser.utils;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Shader;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+
+import androidx.core.content.ContextCompat;
+
+public class DrawableUtils {
+    /**
+     * Creates a circular drawable from a resource drawable ID.
+     *
+     * @param context The Android context
+     * @param drawableResId Resource ID of the drawable to make circular
+     * @param size The desired width and height of the output drawable in pixels
+     * @return A circular BitmapDrawable, or null if the input drawable cannot be loaded
+     */
+    public static Drawable getCircularDrawable(Context context, int drawableResId, int size) {
+        Drawable sourceDrawable = loadSourceDrawable(context, drawableResId);
+        if (sourceDrawable == null) {
+            return null;
+        }
+
+        Bitmap sourceBitmap = drawableToBitmap(sourceDrawable, size, size);
+        return createCircularBitmapDrawable(context, sourceBitmap, size);
+    }
+
+    private static Drawable loadSourceDrawable(Context context, int drawableResId) {
```

Comment:
this method is also only called in one place internally and just calls another static method and does nothing else?